agreement/__manifest__.py

```diff
@@ -17,6 +17,8 @@
         'mail',
         'product',
         'sale_management',
+        'maintenance',
```

Comment:
Please don't add this hard dependencies. You should create extension modules like `agreement_maintenance` for handling this.

Reply:
Make sense. I will be on vacation so will do it when I back. Thank you.